pkg/istio-agent/xds_proxy_test.go

```diff
@@ -284,12 +299,18 @@ func TestXdsProxyReconnects(t *testing.T) {
 
 		conn := setupDownstreamConnection(t, proxy)
 		downstream := stream(t, conn)
-		sendDownstream(t, downstream)
+		sendDownstreamWithNode(t, downstream, model.NodeMetadata{
+			Namespace:   "default",
+			InstanceIPs: []string{"1.1.1.1"},
+		})
 		conn.Close()
 
 		c := setupDownstreamConnection(t, proxy)
 		downstream = stream(t, c)
-		sendDownstream(t, downstream)
+		sendDownstreamWithNode(t, downstream, model.NodeMetadata{
```

Comment:
why did node need to be added?

Reply:
I am not sure. I just moved the existing code as is - earlier sendDownstream is calling this. Now removed that method and made the call inline